CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 * [BUGFIX] Alerts: Fixed `MimirMemoryMapAreasTooHigh` alert when Mimir is deployed in read-write mode. #3626
 * [BUGFIX] Alerts: Fixed `MimirCompactorSkippedBlocksWithOutOfOrderChunks` matching on non-existent label. #3628
 * [BUGFIX] Dashboards: Fix `Rollout Progress` dashboard incorrectly using Gateway metrics when Gateway was not enabled. #3709
+* [BUGFIX] Tenants dashboard: Make it compatible with all deployment types. #xxxx
```

Comment:
```suggestion
* [BUGFIX] Tenants dashboard: Make it compatible with all deployment types. #3754
```

Reply:
You are too fast :) , i was waiting for checks to complete